config/pilot-status.cpf

```diff
@@ -0,0 +1,17 @@
+
+SELECT
+  Machine AS "Worker Node" WIDTH -20
+  State AS State WIDTH -9
+  ifThenElse(State=="Claimed", split(GlobalJobId, "#")[1], "") AS "Payload ID" WIDTH -15
+  ifThenElse(State=="Claimed", split(RemoteUser, "@")[0], "") AS User WIDTH -10
+  ifThenElse(State=="Claimed", split(GlobalJobId, "#")[0], "") AS "Scheduler" WIDTH -31
+  ifThenElse(EnteredCurrentActivity isnt undefined, EnteredCurrentActivity, time()) AS "  Job Runtime" PRINTAS ACTIVITY_TIME
+  GLIDEIN_SiteWMS_JobId AS BatchID WIDTH -10
```

Comment:
It actually shows both.  The first GlobalJobId is the job id of the payload job, ie the job id on the remote submission host.  The jobid on the local CE's batch system (Slurm for example) is shown in the BatchID column.